dist/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
```

Comment:
add the .gitignore file to you code before you first commit to make sure you do not upload your node_modules to github